app/graphql/mutations/create_submission.rb

```diff
@@ -11,11 +11,20 @@ class CreateSubmission < ApplicationQuery
 
     def resolve(_params)
       submission = create_submission
-      notify(
-        :success,
-        I18n.t('shared.notifications.done_exclamation'),
-        I18n.t('mutations.create_submission.success_notification')
-      )
+
+      if submission.passed_at.blank?
+        notify(
+          :success,
+          I18n.t('shared.notifications.done_exclamation'),
+          I18n.t('mutations.create_submission.success_notification')
+        )
+      else
+        notify(
+          :success,
+          I18n.t('shared.notifications.done_exclamation'),
+          I18n.t('mutations.create_submission.form_success_notification')
```

Comment:
DRY: The only thing that changes between these two `notify` calls is the second `I18n.t`. I'd extract that out as something like:

```ruby
description_key = submission.passed? ?  'success_notification' : 'form_success_notification'
```

The remaining code should be the same for both paths.